DaysOfCode/abstractClasses.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-const-assign */
+/* eslint-disable no-control-regex */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable max-classes-per-file */
+let _input = '';
+const _index = 0;
+process.stdin.on('data', (data) => { _input += data; });
+process.stdin.on('end', () => {
+  _input = _input.split(new RegExp('\n'));
+  main();
+});
+function readLine() { return _input[_index++]; }
+
+/* *** Ignore above this line. *** */
+
+class Book {
+  constructor(title, author) {
+    if (this.constructor === Book) {
```

Comment:
what? you might want to use "instance of"

Reply:
My code:

![image](https://user-images.githubusercontent.com/50966087/93962849-fc2be900-fd18-11ea-95b5-d799bdc553e0.png)